multivariable-calculus/manipulating-derivatives.md

```diff
@@ -1 +1,67 @@
-# Manipulating partial derivatives
+# Manipulating Partial Derivatives
+
+## Example:
+
+We want to compute $\Delta U$ as a function of $T$ and $P$, so we need to determine:
+
+$$
+dU = \left( \frac{\partial U}{\partial T} \right)_P dT + \left( \frac{\partial U}{\partial P} \right)_T dP
+$$
+
+***
+
+Given that:
+
+$$
+\alpha_V = \frac{1}{V} \left( \frac{\partial V}{\partial T} \right)_P
+$$
+
+$$
+\kappa_T = -\frac{1}{V} \left( \frac{\partial V}{\partial P} \right)_T
+$$
+
+$$
+c_P = \left( \frac{\partial H}{\partial T} \right)_P
+$$
+
+$$
+dU = T dS - P dV
+$$
+
+$$
+dH = T dS + V dP
+$$
+
+$$
+dG = -S dT + V dP
+$$
+
+Express $dU$ using only the "measurables", $T$, $P$, $V$, $\alpha_V$, $\kappa_T$, and $c_P$.
+
+***
+
+Form the derivatives using the given $dU$, then replace what we don't like:
+
+$$
```

Comment:
Please use the align environment (like shown above) to break some of these equations into multiple lines at the equals signs. That will make it easier to read on smaller format screens.